src/modules/ROOT/pages/changelog/202209/08.json

```diff
@@ -67,7 +67,8 @@
     },
     {
       "what": [
-        "showing invalid role permissions when the db was being dropped and re provisioned again."
+        "the permissions on system pages showing invalid roles.",
+        "If you selected one or more roles, then dropped your database and re-provisioned it, the selected role was still showing selected (because it's in local storage), but it was showing deny for all."
```

Comment:
`...the selected role was still showing selected...` -> `...the previous selection was still active...`

or similar; the original sentence reads off to me.